src/Register.js

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from 'react';
+import './App.css';
+import {sendJson} from './sendJson';
+
+export const Register = () => {
+  //The hooks. We might be able to slim the register/login hooks down to one single hook holding an object.
+  const [emailRegisterState, setEmailRegisterState] = useState('');
+  const [passwordRegisterState, setPasswordRegisterState] = useState('');
+  const [error, setError] = useState('');
+
+  //function to submit request to create new user.
+  const submitNewUser = async e => {
+    e.preventDefault();
+    const result = await sendJson('addPerson', {
+      email: emailRegisterState,
+      password: passwordRegisterState,
+      admin: false
```

Comment:
If the `addPerson` endpoint is open to non-admins, they will be able to make themselves admins. I added a task to fix this. We can ignore it for now.